app/views/layouts/application.haml

```diff
@@ -87,7 +87,7 @@
                 %li= link_to 'Favorites', favorites_path
                 %li= link_to "Post", new_post_path
               = link_to logout_path, method: :delete, id: :"header-logout" do
```

Comment:
Ooh, okay, so apparently this confirm helper has to (1) be inside a `data: { ... }` object, and (2) works on the link since that's where we actually define the logout redirect (this button is a no-op). This should work (apparently I can't do a bulk suggestion on GitHub because there's a deleted line here, which is a bit weird, but applying both suggestions seemed to work locally).

```suggestion
              = link_to logout_path, method: :delete, id: :"header-logout", data: { confirm: 'Press OK to confirm logout' } do
```